lib/honeydew/queue.ex

```diff
@@ -53,6 +53,13 @@ defmodule Honeydew.Queue do
 
   @optional_callbacks handle_call: 3, handle_cast: 2, handle_info: 2
 
+  def child_spec(opts) do
+    %{
+      id: __MODULE__,
```

Comment:
if you register this name, only one queue process will be able to start up per VM.

Reply:
right you are, my bad.